packages/web/integrations/axelar/transfer.tsx

```diff
@@ -175,6 +174,15 @@ const AxelarTransfer: FunctionComponent<
     /** Amount, with decimals. e.g. 1.2 USDC */
     const amount = isWithdraw ? withdrawAmountConfig.amount : depositAmount;
 
+    const { transferFee: transferFeeMinAmount } = useTransferFeeQuery(
+      sourceChain,
+      destChain,
```

Comment:
I think we want to leave this variable named `transferFee`. The min amount is used to disable UI if the amount isn't at least that. It's related but not the same.

Reply:
okay